src/style.css

```
@@ -0,0 +1,29 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
```

```suggestion
* {
  box-sizing: border-box;
}
```
Don't reset the margins and paddings using (*) selector. It applies to all elements, which is really bad for performance. Better to specify the exact elements